utils/test_harness/sysman/src/test_harness_sysman_device.cpp

```diff
@@ -68,6 +68,12 @@ uint32_t get_processes_count(zes_device_handle_t device) {
   return count;
 }
 
+uint32_t get_processes_count(zes_device_handle_t device, ze_result_t &result) {
+  uint32_t count = 0;
+  result = zesDeviceProcessesGetState(device, &count, nullptr);
```

Comment:
Minor clarification: W.r.t the earlier offline discussion we had, does this zesDeviceProcessesGetState invocation also return Unsupported feature for the integrated device or is it only with line no. 108 ?

Reply:
zesDeviceProcessesGetState(device, &count, nullptr) also return unsuppoted for iGPU's.
In the newly added overloaded function if the first invocation of zesDeviceProcessesGetState fails, then i am simply returning at line 98